clients/extension/scripts/popup.js

```diff
@@ -10,6 +10,10 @@ const SettingsController = require('./settings-controller.js');
 const VIEWER_URL = 'https://googlechrome.github.io/lighthouse/viewer/';
 const optionsVisibleClass = 'main--options-visible';
 
+const STRINGS = {
+  localhostErrorMessage: chrome.i18n.getMessage('localhostErrorMessage'),
```

Comment:
`chrome.i18n` still works in firefox? lol

Reply:
yea, it's great

Reply:
wild.